.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,9 +14,14 @@
 - chronos
 - mobile
 
-**Run database migrations (delete if no migration was added)**
+**Run database migrations?**
 YES
 
-**Release notes for users (delete if codebase-only change)**
```

Comment:
I don't think we should remove those "delete if" notes, otherwise people will leave them in which will be very annoying.